### PR TITLE
RELATED: ONE-4310 Describe table resizing triggered by the virtual co…

### PR DESCRIPTION
**Pull request by vlastikcz**
…lumns change

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/231/ : SUCCESS in 2m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/86/ : SUCCESS in 9s

File: docs/pivot_table_component.md [hunk 1]
```diff
@@ -207,6 +207,7 @@ columnSizing: {
 ```
 * The size is calculated based on the content in the header of the column that represents the lowest level of the grouped attributes (see [Grouping](#grouping)). If this is not applicable, the size is calculated based on the content in the header of the column with the measure name and the cells with the measure values.
 * Only the columns that are visible during the initial rendering of the table are automatically resized to fit their content.
+* New columns (i.e. columns shown after the initial render) are resized only if the table is not scrolled. This is useful especially for responsive tables with a dynamic width.
```
Comment (evgeniabogranskaya) [inline docs/pivot_table_component.md]: New columns (that is, those that are shown after the initial render) are resized only if the table is not scrollable. This is useful especially for responsive tables with the dynamic width.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/233/ : SUCCESS in 1m 31s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/88/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://jenkins-master02.na.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/205/ : SUCCESS in 2m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/89/ : SUCCESS in 9s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/235/ : SUCCESS in 1m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/71/ : SUCCESS in 26s

**Pull request merged**